### ISSUE TITLE
I have added key in plugin but map key error generated.

### ISSUE THREAD

**Issue by rvpatel** [rilwis/meta-box#977]
Google Maps API error: MissingKeyMapError https://developers.google.com/maps/documentation/javascript/error-messages#missing-key-map-error
https://maps.google.com/maps/api/js?ver=4.6.1 Line 34

I have added map key on plugn map.php file but display error since last 2 days.
Frontend and backed option map not working.

**Comment by ZeroBeeOne** [comment]
Updating to the latest version was reported to resolve this issue for this person: https://github.com/rilwis/meta-box/issues/943, did you try that?

**Comment by rvpatel** [comment]
I have update files from git and my front code for map but not working now front side.

```
<?php 
                            $args = array(
                                'type'         => 'map',
                                'width'        => '100%', // Map width, default is 640px. Can be '%' or 'px'
                                'height'       => '300px', // Map height, default is 480px. Can be '%' or 'px'
                                'zoom'         => 15,  // Map zoom, default is the value set in admin, and if it's omitted - 14
                                'marker'       => true, // Display marker? Default is 'true',
                                'marker_title' => get_the_title(), // Marker title when hover
                                'info_window'  => '<h5>'.get_the_title().'</h5>'.get_the_excerpt(), // Info window content, can be anything. HTML allowed.
                            );
                            echo rwmb_meta( $prifix.'event_map', $args ); // For current post
                        ?>
```

**Comment by rvpatel** [comment]
\plugins\meta-box\inc\fields\map.php
```
//$google_maps_url = add_query_arg( 'key', $field['api_key'], 'https://maps.google.com/maps/api/js' );
$google_maps_url = add_query_arg( 'key','myapikey added hear', 'https://maps.google.com/maps/api/js' );
```
$field['api_key'] value issue arrived. After replace static value this one working.

**Comment by ZeroBeeOne** [comment]
@rvpatel - I'm not a developer for this plugin unfortunately, just floating around the issue tracker watching my own issues, and your issue looked similar to that one. If it's only a problem in the front end, I would suggest you may be able to workaround the issue in a slightly cleaner way (without needing to directly modify the metabox plugin files) by adding a filter to `rwmb_google_maps_url`, but you'll need to wait for rilwis or funkedgeek (or any other plugin devs) if you want a baked-in solution

**Comment by rilwis** [comment]
It's fixed in 30b5db8 and da5ee28. Please try the Github version.
